build.gradle

```diff
@@ -347,7 +348,6 @@ testClusters.integTest {
 
     // Installs all registered zipArchive dependencies on integTest cluster nodes except security
     configurations.zipArchive.asFileTree.each {
-        if(!it.name.contains("opensearch-security")) {
```

Comment:
Hey @owaiskazi19 instead of new `secureIntegTestPluginArchive` configuration, I assume just changing the `if` statement to `if(it.name.contains("opensearch-security"))` should work. Can you please check?

Reply:
@prudhvigodithi here we have to install all the plugins except security plugin. `zipArchive` would have security plugin in it.

Reply:
Yes, +1 to @owaiskazi19 . Initially we kept it how @prudhvigodithi you mentioned above. But the problem, here we just need security plugin to run integ tests, therefore zip archive tries to find it at the compile time also. Therefore, we need to change the tag to securePluginTestArchive

Reply:
Thanks @vibrantvarun and @owaiskazi19 I'm afraid introducing new configuration `secureIntegTestPluginArchive`  might never be called. Also I noticed  the logic is called in `ext` (extra properties) where as I see similar code in `integTest`, If not needed we can completely remove the in `ext`.
```
testClusters.integTest {
    testDistribution = "ARCHIVE"

    // Optionally install security
    if (System.getProperty("security.enabled") != null && System.getProperty("security.enabled") == "true") {
        configureSecurityPlugin(testClusters.integTest)
    }

    // Installs all registered zipArchive dependencies on integTest cluster nodes except security
    configurations.zipArchive.asFileTree.each {
            plugin(provider(new Callable<RegularFile>(){
                    @Override
                    RegularFile call() throws Exception {
                        return new RegularFile() {
                            @Override
                            File getAsFile() {
                                return it
                            }
                        }
                    }
                })
            )

    }
   ```